django_prbac/decorators.py

```diff
@@ -5,8 +5,8 @@
 
 # Local Imports
 from django.http import Http404
-from django_prbac.exceptions import PermissionDenied
 from django_prbac.utils import has_privilege
+from django_prbac.exceptions import PermissionDenied
```

Comment:
The convention is to order imports alphabetically by module name. Why was this changed?